pages/instructorCourseAdminSharing/instructorCourseAdminSharing.sql

```diff
@@ -0,0 +1,54 @@
+-- BLOCK get_course_sharing_info
+SELECT sharing_name, sharing_id
+FROM pl_courses
+WHERE id = $course_id
+
+
+-- BLOCK select_sharing_sets
+SELECT
+    ss.name,
+    ss.id,
+    jsonb_agg(jsonb_build_object(
+        'course_id', c.id,
+        'short_name', c.short_name --maybe should use c.sharing_name here instead?
```

Comment:
Recommended user interface for showing all that information? I'm not sure how to fit it all in, since this was supposed to just be a list of course names all on the same line. 

The problem is that we have a table full of a bunch of sharing sets, and each one has a list of courses that its shared to listed next to it. If we put too much info it will get cluttered . Maybe we could have the extra info show up on hovering over the course name or something?

Reply:
discussed offline, in the future we should have a pop-up with more info that is shown on hover